content/en/blog/_posts/2022-03-31-Kubernetes-1-24-deprecations-and-removals

```diff
@@ -0,0 +1,121 @@
+favor of stable versions of the same APIs. The major change coming in the Kubernetes 1.24 release 
+is the [removal of Dockershim](https://github.com/kubernetes/enhancements/issues/2221). This will 
+be discussed below and in more depth at release time.  
+
+For an  early look at the changes coming in Kubernetes 1.24, take a look at the in-progress [CHANGELOG](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.24.md).
+
+## A note about Dockershim
+
+It's safe to say that the removal receiving the most attention with the release of Kubernetes 1.24 
+is Dockershim. Dockershim was deprecated in 1.20. As noted in the [Kubernetes 1.20 changelog](https://github.com/kubernetes/kubernetes/blob/master/CHANGELOG/CHANGELOG-1.20.md#deprecation): 
+"Docker support in the kubelet is now deprecated and will be removed in a future release. The kubelet 
+uses a module called "dockershim" which implements CRI support for Docker and it has seen maintenance 
+issues in the Kubernetes community." With the upcoming release of Kubernetes 1.24, the Dockershim will 
+finally be removed. 
+
+In the article [Don't Panic: Kubernetes and Docker](/blog/2020/12/02/dont-panic-kubernetes-and-docker/),
+the authors succinctly captured the change's impact and encouraged users to remain calm: 
+> Docker as an underlying runtime is being deprecated in favor of runtimes that use the
+> Container Runtime Interface (CRI) created for Kubernetes. Docker-produced images
+> will continue to work in your cluster with all runtimes, as they always have.
+
+Several guides have been created with helpful information about migrating from dockershim
+to container runtimes that are directly compatible with Kubernetes. You can find them on the
+[Migrating from dockershim](/docs/tasks/administer-cluster/migrating-from-dockershim/)
+page in the Kubernetes documentation.
+
+For more information about why Kubernetes is moving away from dockershim, check out the aptly 
+named: [Kubernetes is Moving on From Dockershim](/blog/2022/01/07/kubernetes-is-moving-on-from-dockershim/) 
+and the [updated dockershim removal FAQ](/blog/2022/02/17/dockershim-faq/).
+
+Take a look at the [What you need to do to have a working cluster for 1.24](/blog/2022/03/31/ready-for-dockershim-removal/) post to learn about how to ensure your cluster continues to work after upgrading from 1.23 to 1.24. 
+
+## The Kubernetes API removal and deprecation process
+
+Kubernetes contains a large number of components that evolve over time. In some cases, this 
+evolution results in APIs, flags, or entire features, being removed. To prevent users from facing 
+breaking changes, Kubernetes adopted a feature [deprecation policy](/docs/reference/using-api/deprecation-policy/). 
+This policy ensures that stable APIs may only be deprecated when a newer stable version of that 
+same API is available and that APIs have a minimum lifetime depending on their stability level: 
```

Comment:
Are you referring to GA, Beta and Alpha as "stability levels"? If so, you might say, "...minimum lifetime as indicated by the following stability levels:"